bin/describe/describe_workspace.ml

```diff
@@ -542,10 +542,10 @@ module Crawl = struct
 
   (* Builds a workspace description for the provided dune setup and context *)
   let workspace
-    options
-    ({ Dune_rules.Main.contexts = _; scontexts } : Dune_rules.Main.build_system)
-    (context : Context.t)
-    dirs
+        options
+        ({ Dune_rules.Main.contexts = _; scontexts } : Dune_rules.Main.build_system)
+        (context : Context.t)
+        dirs
     : Descr.Workspace.t Memo.t
```

Comment:
A bit undecided on this. It's kind of nice that the arguments are not indented the same amount as the function body (as that makes them bleed together), but the return type still is which is quite inconsistent.